test/unit_tests/arkode/CXX_serial/ark_test_dahlquist_mri_-1.out

```diff
@@ -18,12 +18,12 @@ Testing method ARKODE_MIS_KW3
   stages = 4
   method order (q) = 3
   embedding order (p) = 0
-  c = 0  0.3333333333333333  0.75  1  
+  c = 0  0.333333333333333  0.75  1  
   W[0] = 
-                            0                        0                        0                        0  
-           0.3333333333333333                        0                        0                        0  
-          -0.5208333333333333                   0.9375                        0                        0  
-           0.3541666666666666                  -0.6375       0.5333333333333333                        0  
+      0  0  0  0  
+      0.333333333333333  0  0  0  
+      -0.520833333333333  0.9375  0  0  
+      0.354166666666667  -0.6375  0.533333333333333  0  
```

Comment:
For these matrix outputs we should specify a width or use `e` rather than `g` formatting so the columns align

Reply:
I still need to update the .out files, but the `e` format looks much better. Negative sign misaligns columns by a character.

Reply:
Adding a space between `%` and `.` should fix that problem i.e., `% .`